src/components/timeline/Markers.js

```diff
@@ -25,6 +25,9 @@ import type {
 } from '../../utils/connect';
 import type { ThreadIndex } from '../../types/profile';
 
+// Exported for tests.
+export const MIN_MARKER_WIDTH = 0.3;
```

Comment:
Have you tried different values? I think this works fine with the background is white, but not so well when the background is white (eg: gc markers on top of a big user timing marker). Maybe we could bump up this value to 0.5.

Reply:
This was mainly optimizing for the disk io issues with lots of small markers. I didn't mess around with making the "unknown" markers list more legible. I know people use  that area, but it's kind of a mess. Happy to revisit this if you want.